### PR TITLE
Rename: primitives/sr-std -> primitives/sp-std

### PR DESCRIPTION
**Pull request by kigawas**

### CODE REVIEW

**Comment by bkchr** [comment]
I think we should stop renaming all imports directly and just use `sp-std`.

**Comment by kigawas** [comment]
> I think we should stop renaming all imports directly and just use `sp-std`.

You mean rename the `use rstd` stuff?

**Comment by tomusdrw** [comment]
@bkchr @kigawas 
I would remove the crate renames (`X = { package = "sp-X", ...}`) separately to simplify reviewing. It can also be applied as a script to the entire codebase easily, I think @gnunicorn had some ideas how to do it.

@kigawas `sp-` prefix should be dropped from the folder names, so we should rename to `primitives/std` and the crate name should be `sp-std` (it's `substrate-primitives-std`).

File: frame/balances/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ edition = "2018"
 serde = { version = "1.0.101", optional = true }
 safe-mix = { version = "1.0.0", default-features = false }
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
-rstd = { package = "sp-std", path = "../../primitives/sr-std", default-features = false }
+sp-std = { package = "sp-std", path = "../../primitives/std", default-features = false }
```
Comment (gnunicorn) [inline frame/balances/Cargo.toml]:
```suggestion
sp-std = { path = "../../primitives/std", default-features = false }
```

would recommend we remove the `package = ...` everywhere as well.

File: primitives/sr-arithmetic/fuzzer/Cargo.lock [hunk 2]
```diff
@@ -308,7 +308,7 @@ dependencies = [
 ]
 
 [[package]]
-name = "sr-std"
+name = "std"
```
Comment (gnunicorn) [inline primitives/sr-arithmetic/fuzzer/Cargo.lock]: have you tested whether the fuzzer still works?

**Comment by gnunicorn** [comment]
@bkchr , @tomusdrw moving the discussion to the meta issue.

File: primitives/sr-arithmetic/fuzzer/Cargo.lock (hunk 2 above)
Comment (kigawas) [reply primitives/sr-arithmetic/fuzzer/Cargo.lock]: Looks like this is not in CI

Comment (kigawas) [reply primitives/sr-arithmetic/fuzzer/Cargo.lock]: Already fixed the cargo.lock file, but the ci code still needs to add

**Comment by kigawas** [comment]
full crypto failure will be fixed by #4339

**Pull request merged**